docs/syntax_and_semantics/macros/README.md

```diff
@@ -450,7 +473,7 @@ Notice the variables in the inner macro are not available within the `verbatim`
 
 ## Comments
 
-Macro expressions are evaluated both within comments as well as compilable sections of code. This may be used to provide relevant documentation for expansions:
+Macro expressions are evaluated both within comments as well as compatible sections of code. This may be used to provide relevant documentation for expansions:
```

Comment:
**issue:** I think this wording is intentional.
I understand "compilable sections of code" as code blocks inside a doc comment.